nurseconnect/templates/core/header.html

```diff
@@ -13,7 +13,7 @@
             <nav class="NavBar NavBar--siteHead" role="menubar">
                 <ul class="NavBar-navGroup" role="menu">
                     <li class="NavBar-navItem{% if active == "menu" %} NavBar-navItem--active{% endif %}"
-                        role="presentation">
+                        role="presentation" data-enhanced-nav>
```

Comment:
I found this looking for WAI-ARIA compliant navigation patterns, and this was a recommended role to be assigned to the containing `<li>`.